storage/range.go

```diff
@@ -2028,6 +2043,7 @@ func (r *Range) AdminMerge(args *proto.AdminMergeRequest, reply *proto.AdminMerg
 					UpdatedDesc:    updatedDesc,
 					SubsumedRaftID: subsumedDesc.RaftID,
 				},
+				Intents: []proto.Key{desc1Key, desc2Key},
```

Comment:
Ah, right. The comment in data.proto should say "same store" instead of "same range" then.